website/project/views/contributor.py

```diff
@@ -541,7 +541,7 @@ def notify_added_contributor(node, contributor, auth=None, throttle=None, email_
 
     throttle = throttle or settings.CONTRIBUTOR_ADDED_EMAIL_THROTTLE
     # Email users for projects, or for components where they are not contributors on the parent node.
-    if contributor.is_registered and (isinstance(node, Preprint) or
+    if contributor.is_registered and ((isinstance(node, (Preprint, DraftRegistration))) or
```

Comment:
You have redundant parenthesis here.

Reply:
I don't believe it's redundant because the boolean clause continues onto the next line and concludes with a parenthesis that closes that one. Or are you saying that the parenthesis isn't necessary because Python will evaluate the and-or the same as the and-(or)?